packages/dashboard/src/components/composed/Sidebar/Top/ColorSchemeToggle.tsx

```diff
@@ -0,0 +1,41 @@
+import { ActionIcon, useMantineColorScheme, createStyles } from "@mantine/core";
+import { Moon, Sun } from "react-feather";
+
+const useStyles = createStyles((theme, _params, _getRef) => {
+  const { colors, colorScheme } = theme;
+  return {
+    actionIcon: {
+      color: colorScheme === "dark" ? colors.yellow[4] : colors.blue[6],
+      backgroundColor:
+        colorScheme === "dark"
+          ? colors["truffle-brown"][6]
+          : colors["truffle-beige"][2],
+      "&:hover": {
+        backgroundColor:
+          colorScheme === "dark"
+            ? colors["truffle-brown"][7]
+            : colors["truffle-beige"][3]
+      }
+    }
+  };
+});
+
+function ColorSchemeToggle(): JSX.Element {
+  const { classes } = useStyles();
+  const { colorScheme, toggleColorScheme } = useMantineColorScheme();
+  const featherIconProps = { size: 16 };
+
+  const handleClick = () => void toggleColorScheme();
```

Comment:
How do I switch back to system theme?

Reply:
What if we just remove it from local storage and default to system theme upon page load? It's okay if there's no way to unset something if a refresh serves as a painless way to do the same thing!

Reply:
A ternary toggle would probably be confusing. Users might think: I'm setting it to light... dark... and woah! Did I just unset it? 🤔

Anyway I think it might be better to keep it in local storage for now, even though there's no UI to reset it to system. If someone wants the dashboard to have the opposite color scheme than their system color scheme, they'd have to toggle it every time the page loads!

Would it be better, in the future when there's a settings panel / modal of some kind, to have a button there to reset it to system?

Reply:
We'll figure this out later :)